Onboarding/onboarding-schedule-template.md

```diff
@@ -0,0 +1,43 @@
+# Ondboarding schedule template
+
+## Post onboarding (1 week before)
+- Monday
+    - Send welcome email (this should include the intake questionaire and things they should expect going forward)
+    - Create a place to store all relavent documents for the team we are supporting
+    - Send invatations for onboarding meetings
+    - Send list of contact information about the support team
+- Tuesday
+    - Setup Slack accounts and invites
+    - Send instruction for setting up GitHub
+- Wednesday
+    - Connect GitHub account to any repos that are needed
+    - Setup ZenHub for team
+
+## Onboarding week
+- Monday
+    - Onboarding meeting/Kickoff meeting
+- Tuesday
+    - Engineering meeting 
+    - Design meeting
+    - Product meeting 
+- Wednesday 
+    - Meeting to make sure the team is setup and ready for development
```

Comment:
I'm working on expanding this meeting definition here: https://github.com/department-of-veterans-affairs/va.gov-vfs-teams/pull/20

I think we could cross-link these when we get them approved.